src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/preview-tools/panel.xml

```diff
@@ -16,67 +16,74 @@
   -->
 
 <!--
-    This file configures the Preview Tools right panel. The structure of this file follows:
-
-    <panel>
-        <modules>
-            <module>
-                <moduleName />
-                <title />
-                <config>
-                    (module specific configuration)
-                </config>
-            </module>
-        </modules>
-    </panel>
-
-    For the module: Medium Panel (aka Publishing Channel), the configuration follows:
-    <channels>
-        <channel>      (target device)
-            <title />
-            <value />  (this is the browser's UserAgent)
-            <width />  (width in pixels)
-            <height /> (height in pixels)
-        </channel>
-    </channels>
+- - - - - - - - - - - - 
```

Comment:
This whole file is not used anymore (in PageBuilder). Let's leave as it was (rollback)